ft-sdk/src/auth/provider.rs

```diff
@@ -169,6 +169,34 @@ pub fn user_data_by_identity(
     Ok((id, data))
 }
 
+pub fn user_data_by_id(
+    conn: &mut ft_sdk::Connection,
+    provider_id: &str,
+    user_id: &ft_sdk::UserId,
+) -> Result<(ft_sdk::auth::UserId, ft_sdk::auth::ProviderData), ft_sdk::auth::UserDataError> {
```

Comment:
The `UserID` in input and output params are different, use the same (general code cleanliness). Also why are we returning `UserId` at all, why not just the `ProviderData`?

Reply:
I've removed returned `UserId` from this function in https://github.com/FifthTry/ft-sdk/pull/52/commits/b0261d529c653376cd679de0836abada9a059be9